src/MassTransit/Configuration/Registration/RegistrationConfigurator.cs

```diff
@@ -11,26 +11,23 @@ namespace MassTransit.Registration
     using Internals.Extensions;
     using Metadata;
     using Microsoft.Extensions.Logging;
-    using Monitoring.Health;
     using Saga;
 
 
     /// <summary>
-    /// Used for registration of consumers and sagas
+    ///     Used for registration of consumers and sagas
     /// </summary>
     public class RegistrationConfigurator :
         IRegistrationConfigurator
     {
-        readonly IContainerRegistrar _containerRegistrar;
+        readonly ConcurrentDictionary<Type, IActivityRegistration> _activityRegistrations;
```

Comment:
Not at all related to this PR, but I can't understand why I used ConcurrentDictionary for something that is clearly synchronous.